src/utils/constants.js

```diff
@@ -1,6 +1,7 @@
 export const CONTRACT_ADDRESS = "0xA521Fb5BC2ffbA545c7f1202370dbA10fb410312";
 export const WETH_ADDRESS = "0xd0A1E359811322d97991E03f863a0C30C2cF029C";
 export const USDC_ADDRESS = "0xe22da380ee6B445bb8273C81944ADEB6E8450422";
+export const COINBASEAPI="https://api.coinbase.com/v2/prices/ETH-USD/spot"
```

Comment:
COINBASEAPI -> COIN_BASE_ASSET_PRICE_URL
or add a comment stating what the API returns "USD Price of an asset"